python/servo/package_commands.py

```diff
@@ -105,6 +105,32 @@ def copy_dependencies(binary_path, lib_path):
         need_checked.difference_update(checked)
 
 
+def change_prefs(resources_path, platform):
+    prefs_path = path.join(resources_path, "prefs.json")
+    package_prefs_path = path.join(resources_path, "package-prefs.json")
+    os_type = "os:{}".format(platform)
+    with open(prefs_path) as prefs:
+        prefs = json.loads(prefs.read())
+        with open(package_prefs_path) as package_prefs:
+            package_prefs = json.loads(package_prefs.read())
```

Comment:
Note that there's `json.load` (just like `json.dump`) that takes a file descriptor and parses the contents to a JSON object directly.